tools/wasmedge/wasmedgec.cpp

```diff
@@ -155,9 +155,9 @@ int main(int Argc, const char *Argv[]) {
     if (ConfGenericBinary.value()) {
       Conf.getCompilerConfigure().setGenericBinary(true);
     }
-    if (OutputPath.extension().u8string() == ".wasm"sv) {
+    if (OutputPath.extension().u8string() == ".so"sv) {
```

Comment:
Looks like this is a breaking change in wasmedgec. We should add it on changelog.